modules/log6x/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="cluster-logging-collector-log-forward-kafka_{context}"]
+= Forwarding logs to a Kafka broker
+
+You can forward logs to an external Kafka broker in addition to, or instead of, the default log store.
+
+To configure log forwarding to an external Kafka instance, you must create a `ClusterLogForwarder` custom resource (CR) with an output to that instance, and a pipeline that uses the output. You can include a specific Kafka topic in the output or use the default. The Kafka output can use a TCP (insecure) or TLS (secure TCP) connection.
+
+.Procedure
+
+. Create or edit a YAML file that defines the `ClusterLogForwarder` CR object:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: <log_forwarder_name> <1>
+  namespace: <log_forwarder_namespace> <2>
+spec:
+  serviceAccountName: <service_account_name> <3>
+  outputs:
+   - name: app-logs <4>
```

Comment:
- kafka:
      authentication:
        sasl:
          mechanism: SCRAM-SHA-512
          password:
            key: password
            secretName: to-kafka-secret
          username:
            key: username
            secretName: to-kafka-secret
      url: tls://${amq_url}:443/${topic_name}
    name: amq-kafka
    tls:
      ca:
        key: ca-bundle.crt
        secretName: to-kafka-secret
    type: kafka